packages/babel-generator/test/fixtures/minified/literals/actual.js

```diff
@@ -4,6 +4,7 @@
 '\x20';
 "\n\r";
 "😂";
+`😂`;
```

Comment:
I’ve added this test to indicate that this patch only handles string literals and not template literals. (Regular expression literals are already taken care of through our use of [regexpu](https://mths.be/regexpu).)